src/Ocelot.Provider.Polly/PollyQoSResiliencePipelineProvider.cs

```diff
@@ -0,0 +1,96 @@
+using Ocelot.Configuration;
+using Ocelot.Logging;
+using Ocelot.Provider.Polly.Interfaces;
+using Polly.CircuitBreaker;
+using Polly.Registry;
+using Polly.Timeout;
+using System.Net;
+
+namespace Ocelot.Provider.Polly;
+
+public class PollyQoSResiliencePipelineProvider : IPollyQoSResiliencePipelineProvider<HttpResponseMessage>
```

Comment:
No XML docs which must explain to developer that this is new v8 version!

Reply:
Fixed in commit https://github.com/ThreeMammals/Ocelot/pull/1914/commits/c513b98ff89685fd9ad706d6c1e6b91e879cc652